6-datascience/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:6.10
```

Comment:
This will maintain itself more easily if you change this line to `FROM node:6`. Then the lower layers can be updated automatically with 6.x bugfixes and security patches.